<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfiguration</class>
 <widget class="QDialog" name="DialogConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration de la connexion</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonTester">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tester la connexion</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>DÃ©bit</string>
   </property>
   <widget class="QRadioButton" name="radio4800">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>4800</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio38400">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>38400</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio9600">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>9600</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio19200">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>19200</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio28800">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>28800</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio57600">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>57600</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio115200">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>115200</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
